addons/skin.confluence/720p/DialogPVRGuideOSD.xml

```diff
@@ -203,7 +219,15 @@
 						<width>20</width>
 						<height>20</height>
 						<texture>PVR-HasTimer.png</texture>
-						<visible>ListItem.HasTimer + !ListItem.HasTimerSchedule + !ListItem.IsRecording</visible>
+						<visible>ListItem.HasTimer + !ListItem.HasTimerSchedule + !ListItem.IsRecording + ListItem.IsEnable</visible>
+					</control>
+					<control type="image">
+						<left>120</left>
+						<top>5</top>
+						<width>20</width>
+						<height>20</height>
+						<texture>PVR-HasTimer-Disabled.png</texture>
+						<visible>ListItem.HasTimer + !ListItem.HasTimerSchedule + !ListItem.IsRecording + !ListItem.IsEnabled</visible>
 					</control>
```

Comment:
@ronie Is it somehow possible to add all these PVR-xy icons in one include or something? I'm just copying all this same to all different pvr windows and it doesn't feel right :p